includes/abbreviations.md

```diff
@@ -1,3 +1,2 @@
-# Abbreviations
```

Comment:
I need to remove this otherwise there will be the headline appended at the end of each page